include/hal.h

```diff
@@ -29,11 +29,15 @@
 #include "target.h"
 
 /* Architecture specific calls */
+
```

Comment:
Only whitespace changes in this file... Might consider reverting.

Reply:
fixed